sdk/helper/ldaputil/config.go

```diff
@@ -18,6 +19,11 @@ import (
 // Not all fields will be used by every integration.
 func ConfigFields() map[string]*framework.FieldSchema {
 	return map[string]*framework.FieldSchema{
+		"anonymous_group_search": {
+			Type:        framework.TypeBool,
+			Default:     false,
+			Description: "Use anonymous binds when performing LDAP group searches",
```

Comment:
I think it makes sense to add that the credentials are stilled used for the initial connection test otherwise users could be confused.

Reply:
Can do!